src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeSignalConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by Bansi Chapla on 23/11/2021.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeSignalConfiguration implements IPartialUpdateEntity {
```

Comment:
"enabledAt" field is missing

Reply:
enabledAt should not be used. This field is internal field which converts other param values to some hash code. So this field shouldn't be used.